addons/mail/static/src/discuss/call/public_web/discuss_app/sidebar/call_participants.scss

```diff
@@ -45,3 +45,7 @@
         padding: map-get($spacers, 1) / 2;
     }
 }
+
+.o-mail-DiscussSidebarCallParticipants-avatarStack {
+    margin: 1px;
+}
```

Comment:
Note: used to be within the avatar stack to prevent flickering when switching from stack to list. It is only needed by the `call_participants` component and should not impact other stacks.